cpp/Esri/ArcGISRuntime/Toolkit/CoordinateConversionOption.cpp

```diff
@@ -0,0 +1,257 @@
+/*******************************************************************************
+ *  Copyright 2012-2020 Esri
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ******************************************************************************/
+#include "CoordinateConversionOption.h"
+
+// ArcGISRuntime headers
+#include <CoordinateFormatter.h>
+#include <Point.h>
+
+namespace Esri
+{
+namespace ArcGISRuntime
+{
+namespace Toolkit
+{
+
+CoordinateConversionOption::CoordinateConversionOption(QObject* parent):
+  QObject(parent),
+  m_outputMode(CoordinateType::Usng),
+  m_addSpaces(true),
+  m_precision(8),
+  m_decimalPlaces(6),
+  m_mgrsConversionMode(MgrsConversionMode::Automatic),
+  m_latLonFormat(LatitudeLongitudeFormat::DecimalDegrees),
+  m_utmConversionMode(UtmConversionMode::LatitudeBandIndicators),
+  m_garsConvesrionMode(GarsConversionMode::Center)
+{
+}
+
+/*!
+   \brief The destructor.
+ */
+CoordinateConversionOption::~CoordinateConversionOption()
+{
+}
+
+CoordinateConversionOption::CoordinateType CoordinateConversionOption::outputMode() const
+{
+  return m_outputMode;
+}
+
+void CoordinateConversionOption::setOutputMode(CoordinateType outputMode)
+{
+  if (outputMode == m_outputMode)
+    return;
+
+  m_outputMode = outputMode;
+  emit outputModeChanged();
+}
+
+QString CoordinateConversionOption::name() const
+{
+  return m_name;
+}
+
+void CoordinateConversionOption::setName(const QString& name)
+{
+  if (name == m_name)
+    return;
+
+  m_name = name;
+  emit nameChanged();
+}
+
+bool CoordinateConversionOption::addSpaces() const
```

Comment:
I guess this should be `isAddSpaces` - currently it sounds like it will actually perform an add